packages/block-editor/src/store/reducer.js

```diff
@@ -1043,6 +1043,25 @@ export function blockSelection( state = BLOCK_SELECTION_INITIAL_STATE, action )
 	return state;
 }
 
+/**
+ * Reducer returning the block zoom's state.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export function blockZoom( state, action ) {
```

Comment:
Should the blockZoom reduce handle the case where a sibling of the zoomed blocks is inserted?
E.g: If add a paragraph, zoom it and then press enter the editor crashes.

Reply:
Yes, definitely.